### PR TITLE
Better settings interface

### PR DESCRIPTION
**Pull request by micahlt**
To see these changes, use the latest version of Itchy from Google Play.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -219,6 +219,33 @@ async function unifiedSearch(query, offset) {
...
+function getPrefs() {
+  let prefs = window.localStorage.getItem("preferences");
+  prefs = JSON.parse(prefs);
+  function init() {
+    prefs = initialPrefs;
+    window.localStorage.setItem("preferences", JSON.stringify(prefs));
+    //window.location.reload();
+  }
+  if (prefs) {
+    Object.keys(initialPrefs).forEach((pref) => {
```
Comment (locness3) [inline src/utils.js:237]: I have changed this as well.

**Comment by micahlt** [comment]
> > To see these changes, use the latest version of Itchy from Google Play.
> 
> so you push updates to google play before providing source code in master?

This was an accident - I didn't realize that I hadn't merged changes.

**Comment by locness3** [comment]
What a huge mess, please make sure to work from latest commit to master in the future.

**Comment by locness3** [comment]
Wait, I didn't get notified for this PR before making mine, so I thought this one was made afterwards. It's fine.

**Pull request merged**